reconcile/utils/openshift_resource.py

```diff
@@ -280,6 +293,13 @@ def sha256sum(self):
         annotations = body['metadata']['annotations']
         return annotations['qontract.sha256sum']
 
+    def expiration_date(self):
+        body = self.annotate().body
+
+        # annotations = body['expirationDate']
+        # return annotations['expirationDate']
+        return body['expirationDate']
+
```

Comment:
the class `OpenshiftResource` really represents Kubernetes objects and not the schema object from qontract-schema. therefore `expirationDate` does not exist within this context.

please have a closer look at this class. it represents Kubernetes objects in a very generic way, so not specific to any concrete K8S type. having handling for a very specific type (e.g. Role) in here does not fit well the purpose of this class.

the `expirationDate` only exists within qontract-schema, so please move the methods/functions `expiration_date`,  `has_valid_expiration_date`, `has_expiration_date`, `role_still_valid` and the `EXPIRATION_DATE` constant to a more appropriate context. how about `reconcile/openshift_rolebindings.py` or `reconcile/openshift_groups.py`?